docs/index.html

```diff
@@ -420,7 +420,7 @@ <h4 class="code-container-style"><b>View stats of your hosted challenge with ID
                 </div>
                 <p id="footer-margin">
                     Maintained by the EvalAI core team
-                    <br> ©2019 EvalAI
+                    &copy;<script>new Date().getFullYear()&&document.write(new Date().getFullYear());</script> EvalA
```

Comment:
Change `EvalA` to `EvalAI`

Reply:
Done. So irresponsible of me to forget it. Sorry mate